scripts/spi_flash/board_defs.py

```diff
@@ -128,6 +128,15 @@
         'mcu': "stm32g0b1xx",
         'spi_bus': "spi1",
         "cs_pin": "PB8"
+    },
+    'kingroon-kp3-v1.3-board': {
+        'mcu': "stm32f103xe",
+        'spi_bus': "swspi",
+        'spi_pins': "PC8,PD2,PC12",
+        "cs_pin": "PC11",
```

Comment:
It would be good if you could rebase this on master again, including the change to the conversion script.  Which will do away with the 'check_need_convert' change you have made in spi_flash.py also.

If you could try this with the sdio method that I have in PR https://github.com/Klipper3d/klipper/pull/6679 it would be appreciated also.  The SDIO mode means that it can do the verification also, so is a nicer usage flow if possible.